### PR TITLE
Fix js errors

### LINKED ISSUE #4601: Sync now! problem - Uncaught ReferenceError: doRequest is not defined
`Sync now!` problem - `Uncaught ReferenceError: doRequest is not defined`

**Branch**: 0.15 - Windows installer compiled on 10/9 on both WXP & W7  

**Expected Behavior**: Pressing the `Sync now!` link under `Devices > Available actions` in the `Manage > Facilities` tab should start the syncing process.

**Current Behavior**: Pressing the button has no visible effect for the user, and throws an error:

**Windows 7**

sync-now

Sync progress is visible if one follows the link under **Device name**. 

Also, on first attempt to see the sync process on **WXP** I did get an error, though in resolved on the following server restart:

sync2

### LINKED ISSUE #4602: Check answer problem - Uncaught TypeError: Cannot read property 'response_log' of undefined
Check answer problem - `Uncaught TypeError: Cannot read property 'response_log' of undefined`

**Branch**: 0.15 - Windows installer compiled on 10/9 on both WXP & W7

**Current Behavior**: Pressing the **Check answer** button in the exercise panel throws an error:

check_answer

In addition there is a problem in the **Show hint** button (the highlighted area), as the value for number of hints left is not correctly displayed.

### PR DESCRIPTION
**Pull request by rtibbles**
Fixes #4601, resolves #4602 parts 1 and 2.

### CODE REVIEW

File: kalite/distributed/static/js/distributed/exercises/perseus-helpers.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ require("../perseus/lib/mathquill/mathquill.css");
 require("../perseus/lib/kas.js");
 global.Jed = require("jed");
 require("../perseus/ke/local-only/i18n");
-global.$._ = gettext;
```
Comment (aronasorman) [inline kalite/distributed/static/js/distributed/exercises/perseus-helpers.js]: How do we get the right `gettext` function now?

File: kalite/control_panel/static/js/control_panel/utils/force_sync.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
+var api = require("utils/api");
+
```
Comment (MCGallaspy) [inline kalite/control_panel/static/js/control_panel/utils/force_sync.js]: Needs `var sprintf = require("sprintf-js").sprintf;` for line 9 below.

Comment (MCGallaspy) [inline kalite/control_panel/static/js/control_panel/utils/force_sync.js]: Also needs a `require` stmt for `show_message` below.

File: kalite/distributed/static/js/distributed/perseus/ke/local-only/i18n.js [hunk 3]
```diff
@@ -104,6 +104,8 @@
             str = str.messages[0];
         }
 
+        str = gettext(str);
```
Comment (rtibbles) [inline kalite/distributed/static/js/distributed/perseus/ke/local-only/i18n.js]: @aronasorman See here.

**Comment by MCGallaspy** [comment]
Transaction management error, our old friend.

**Comment by MCGallaspy** [comment]
However, the fix works.

**Pull request merged**